src/main/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/SecondaryNaicsList.java

```diff
@@ -12,15 +12,15 @@
 import jakarta.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
 
 @XmlAccessorType(XmlAccessType.FIELD)
-@XmlType(name = "", propOrder = { "secondaryNaics" })
+@XmlType(name = "", propOrder = { "secondaryNaicItems" })
 @XmlRootElement(name = "SecondaryNaicsList")
-public class SecondaryNaicsList {
+public class SecondaryNaicsList implements NaicsCodeListItem {
 
     @XmlAttribute(name = "isChanged")
     @XmlJavaTypeAdapter(CollapsedStringAdapter.class)
     private String isChanged;
     @XmlElement(name = "SecondaryNaics")
-    private List<SecondaryNaics> secondaryNaics;
+    private List<SecondaryNaicsItem> secondaryNaicItems;
```

Comment:
Many of your other objects are still keeping the "s" in "naics". Should this property be named "secondaryNaicsItems" instead for consistency?

Reply:
ah, yes, I will fix